routes/users.js

```diff
@@ -0,0 +1,57 @@
+const express = require("express");
+const router = express.Router();
+const createError = require("http-errors");
+const admin = require("firebase-admin");
+
+const User = require("../models/User");
+
+const MESSAGE = require("../constants/messages");
+const ERROR = require("../constants/error");
+
+router.get("/check_member", async (req, res, next) => {
+  const { EXPIRED_TOKEN, UNEXPECTED_ERROR, OK } = MESSAGE;
+  const { AUTH_ID_TOKEN_REVOKED, AUTH_ID_TOKEN_EXPIRED } = ERROR;
```

Comment:
스타일 통일을 위해 다음과 같이 수정 부탁드립니다.

```suggestion
  const {
    EXPIRED_TOKEN,
    UNEXPECTED_ERROR,
    OK,
  } = MESSAGES;
  
  const {
    AUTH_ID_TOKEN_REVOKED,
    AUTH_ID_TOKEN_EXPIRED,
  } = ERRORS;
```